src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3) {
+        return a + b;
+      } else if (counter % 2 === 0) {
+        return 'Bzzz... Error!';
```

Comment:
you can just move return Error out of if statement - it will be reached eventually, or function will return before that - else isn't needed